components/persistor/lib/knex/db.ts

```diff
@@ -892,6 +892,14 @@ module.exports = function (PersistObjectTemplate) {
         return map;
     }
 
+    function generateNotifyQueries(template, queriesToNotify, query) {
```

Comment:
Can we have some types or params as to what is going on in this function?

Reply:
this is an internal function, I can type second and third parameter, but first one is a supertype, may be we can consider this as a bigger refactor.